### PR TITLE
Create test cases for AdvanceCommand

### PR DESCRIPTION
**Pull request by bofeng1999**
## What
Add test cases for `AdvanceCommandTest` class.

## Why
Insufficient test cases for the above mentioned classes and changes were made to enhance code quality.

This ensures that the codebase is operating according to the intended business logic.

## How
Added test cases for public methods in the `AdvanceCommand` class.

Fix: https://github.com/AY2223S2-CS2103T-W14-4/tp/issues/42

### CODE REVIEW

**Review by FredericChow00 — APPROVED** [review]
LGTM!

File: src/test/java/seedu/address/logic/commands/AdvanceCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    public void equals() {
+        Person alice = new PersonBuilder().withName("Alice").build();
+        Person bob = new PersonBuilder().withName("Bob").build();
+        NamePhoneNumberPredicate alicePredicate = new NamePhoneNumberPredicate(alice.getName(), alice.getPhone());
+        NamePhoneNumberPredicate bobPredicate = new NamePhoneNumberPredicate(bob.getName(), bob.getPhone());
+        AdvanceCommand advanceFirstCommand = new AdvanceCommand(alicePredicate, Optional.empty());
+        AdvanceCommand advanceSecondCommand = new AdvanceCommand(bobPredicate, Optional.empty());
+
+        // same object -> returns true
+        assertTrue(advanceFirstCommand.equals(advanceFirstCommand));
```
Comment (Merrickneo) [inline src/test/java/seedu/address/logic/commands/AdvanceCommandTest.java:121]: Great work in checking for the different edge cases

**Review by Merrickneo — APPROVED** [review]
LGTM! Test cases looks robust and complete.
